File: app/src/main/java/org/mifos/mobile/ui/views/TransferProcessView.java
package org.mifos.mobile.ui.views;

import org.mifos.mobile.ui.views.base.MVPView;

/**
 * Created by dilpreet on 1/7/17.
 */

public interface TransferProcessView extends MVPView {

    void showTransferredSuccessfully();

    void showError(String msg);
}

File: app/src/main/java/org/mifos/mobile/ui/views/TransferProcessView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.ui.views.base.MVPView

/**
 * Created by dilpreet on 1/7/17.
 */
interface TransferProcessView : MVPView {
    fun showTransferredSuccessfully()
    fun showError(msg: String?)
}